src/plugins/condition/ConditionManager.js

```diff
@@ -32,6 +32,10 @@ export default class ConditionManager extends EventEmitter {
         this.timeAPI = this.openmct.time;
         this.latestTimestamp = {};
         this.initialize();
+
+        this.stopObservingForChanges = this.openmct.objects.observe(this.conditionSetDomainObject, '*', (newDomainObject) => {
```

Comment:
The problem with this is that everytime the conditionManager persist the conditionCollection it's going to trigger the observe and do the update. That's a lot of thrashing.
And doing this won't fix the issue with the telemetry not updating in the conditionSet view when the conditionSet is being actively changed. (Have you tested this scenario?)

Reply:
expensive updates to be fixed in #2750